recipes/lemur/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "lemur" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: 'https://github.com/treangenlab/lemur/archive/refs/tags/v{{version}}.tar.gz'
+  sha256: 591cc3d1d8a7ebb8b51486cb9c74b61dc39aeb8d8e3baf4d318e04278f9f6459
+
+build:
+  noarch: generic
+  number: 0
+  script: mkdir -p ${PREFIX}/bin && cp lemur ${PREFIX}/bin
```

Comment:
might need to add a `chmod` here as well

Reply:
The original file is executable. Looking at the lint error i see
```
ERROR: recipes/lemur/meta.yaml:0: missing_run_exports: Recipe should have a run_exports statement that ensures correct pinning in downstream packages
```